mobilab/settings.py

```diff
@@ -11,6 +11,11 @@
 """
 
 from pathlib import Path
+from json import load
```

Comment:
Everything its looking good so far, as I said before we can opted to use html templates.